Example/Highlightr.xcodeproj/project.pbxproj

```diff
@@ -603,6 +485,7 @@
 			baseConfigurationReference = 4CF75D031FDF606D266D5CFE /* Pods-Highlightr_Example.debug.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				DEVELOPMENT_TEAM = RV92JH4QP5;
```

Comment:
Please remove your development team

Reply:
Hi! I thought I had removed it but forgot to do so in the Example Project. Should be fixed now.

03dd0f36334ee12d8215e52290b1fc617d255b80